offchain/hello-world-browser/src/Main.purs

```diff
@@ -83,7 +83,7 @@ main =
           , networkId = networkId
           }
       false -> do
-        pure $ testnetNamiConfig { logLevel = Warn }
+        pure $ testnetNamiConfig { logLevel = Debug }
```

Comment:
NITPICK: is this for development purpose? should we change it back to `Warn`?

Reply:
Yup, my bad, thanks for catching that